karate-core/src/main/java/com/intuit/karate/http/HttpRequestBuilder.java

```diff
@@ -183,7 +179,7 @@ public HttpRequest build() {
             request.setBodyForDisplay(multiPart.getBodyForDisplay());
         }
         if (cookies != null && !cookies.isEmpty()) {
-            List<String> cookieValues = new ArrayList(cookies.size());
+            List<String> cookieValues = new ArrayList<>(cookies.size());
```

Comment:
I know it is just me or the style I follow but can we lose the un-necessary angle-brackets